client/src/components/SupermarketDisplay.js

```diff
@@ -19,7 +19,7 @@ SupermarketDisplay.propTypes = {
     name: PropTypes.string,
     matchingProductsFound: PropTypes.number,
     listProductCount: PropTypes.number,
-    totalPrice: PropTypes.number,
+    totalPrice: PropTypes.string,
```

Comment:
idk if it is a good practice to express a price in a string. Perhaps you can find a better was to do it in here: [https://frontstuff.io/how-to-handle-monetary-values-in-javascript](https://frontstuff.io/how-to-handle-monetary-values-in-javascript)

Reply:
The price is a string as it includes the € symbol as format